pkg/sql/lease_internal_test.go

```diff
@@ -268,7 +268,9 @@ CREATE TABLE t.test (k CHAR PRIMARY KEY, v CHAR);
 // Tests that a name cache entry with by an expired lease is not returned.
 func TestNameCacheEntryDoesntReturnExpiredLease(t *testing.T) {
 	defer leaktest.AfterTest(t)()
-	s, db, kvDB := serverutils.StartServer(t, base.TestServerArgs{})
+	s, db, kvDB := serverutils.StartServer(t, base.TestServerArgs{
+		MaxOffset: 10 * LeaseDuration, // so we can advance the clock
```

Comment:
I believe you should just use math.MaxInt64 here. We should document this as a way to disable the use of MaxOffset